system/db/features.txt

```diff
@@ -105,6 +105,9 @@
 		
 		// Changes Hasisas to not base crit bonus on HP lost
 		{ name: "HasisasNoHpCritBonus", enabled: true },
+
+		// Changes Pierce to not base its hit count on target size
```

Comment:
Just for clarification, this was an official change that occurred some time after Re:Build was added?

Reply:
I believe so.  I can't find the exact patch where they changed it, but as of 2019 the patch notes still list it doing 2-4 hits.